### PR TITLE
updated group/shifted norml2 tests, attempt 2

### PR DESCRIPTION
**Pull request by rjbaraldi**

### CODE REVIEW

File: src/shiftedGroupNormL2.jl [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  h::NormL2{R},
+  xk::AbstractVector{R}
+) where {R <: Real} = ShiftedGroupNormL2(GroupNormL2([h.lambda]), xk, zero(xk), false)
+shifted(
+  ψ::ShiftedGroupNormL2{R, RR, I, V0, V1, V2},
+  sj::AbstractVector{R},
+) where {R <: Real, RR <:  AbstractVector{R}, I, V0 <: AbstractVector{R}, V1 <: AbstractVector{R}, V2 <: AbstractVector{R}} =
+ShiftedGroupNormL2(ψ.h, ψ.xk, sj, true)
+
+fun_name(ψ::ShiftedGroupNormL2) = "shifted ∑ᵢ||⋅||₂ norm"
```
Comment (dpo) [inline src/shiftedGroupNormL2.jl:42]:
```suggestion
fun_name(ψ::ShiftedGroupNormL2) = "shifted ∑ᵢ‖⋅‖₂ norm"
```
```suggestion
fun_name(ψ::ShiftedGroupNormL2) = "shifted ∑ᵢ||⋅||₂ norm"
```

File: src/shiftedGroupNormL2.jl [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+end
+
+shifted(
+  h::GroupNormL2{R,RR, I},
+  xk::AbstractVector{R},
+) where {R <: Real, RR <: AbstractVector{R}, I} = ShiftedGroupNormL2(h, xk, zero(xk), false)
+shifted(
+  h::NormL2{R},
+  xk::AbstractVector{R}
+) where {R <: Real} = ShiftedGroupNormL2(GroupNormL2([h.lambda]), xk, zero(xk), false)
```
Comment (dpo) [inline src/shiftedGroupNormL2.jl:35]: Nice!

**Review by dpo — COMMENTED** [review]
Just a dumb comment

**Comment by rjbaraldi** [comment]
Didn't miss anything, did I?

**Comment by dpo** [comment]
Thank you, sir!

**Pull request merged**